### PR TITLE
more kotlin

### PR DESCRIPTION
**Pull request by jaredsburrows**

### CODE REVIEW

File: api/src/main/kotlin/com/github/benmanes/gradle/versions/updates/gradle/GradleUpdateChecker.kt [hunk 1]
```diff
@@ -20,7 +20,7 @@ class GradleUpdateChecker(
    * @return if the check for Gradle updates was enabled and, if so, the versions were fetched.
    */
   @get:JvmName("isEnabled")
-  val enabled: Boolean = true,
+  val enabled: Boolean,
```
Comment (ben-manes) [inline api/src/main/kotlin/com/github/benmanes/gradle/versions/updates/gradle/GradleUpdateChecker.kt:23]: this changes the default to false?

**Review by ben-manes — APPROVED** [review]
Please make sure that gradle update checks are enabled by default

**Comment by jaredsburrows** [comment]
Closing this in favor of https://github.com/ben-manes/gradle-versions-plugin/pull/627.

File: api/src/main/kotlin/com/github/benmanes/gradle/versions/updates/gradle/GradleUpdateChecker.kt (hunk 1 above)
Comment (jaredsburrows) [reply api/src/main/kotlin/com/github/benmanes/gradle/versions/updates/gradle/GradleUpdateChecker.kt:23]: This originally had no value - https://github.com/ben-manes/gradle-versions-plugin/blob/c2bad3d8661e203410c80afddcbb3ade4fdc772e/src/main/groovy/com/github/benmanes/gradle/versions/updates/gradle/GradleUpdateChecker.groovy#L26.

I can keep it `true`.

Comment (jaredsburrows) [reply api/src/main/kotlin/com/github/benmanes/gradle/versions/updates/gradle/GradleUpdateChecker.kt:23]: https://github.com/ben-manes/gradle-versions-plugin/blob/9f0260e419dc43be340bc048890eef1aaabfd71a/src/main/groovy/com/github/benmanes/gradle/versions/updates/DependencyUpdates.groovy#L49

**Pull request closed without merging**